bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java

```diff
@@ -555,7 +553,7 @@ void encode(byte[] data, long b, int offset, int len) {
         }
 
         byte[] encodePing() {
-            // MSG 0x00000065 0x0e414e47 f 0x00000001 0xbc2fbb5f 0x00000062 PIN    � ��� ��    R e g u l a rFABE
+            // MSG 0x00000065 0x0e414e47 f 0x00000001 0xbc2fbb5f 0x00000062 
```

Comment:
Please remove the trailing space or the check will fail.

```suggestion
            // MSG 0x00000065 0x0e414e47 f 0x00000001 0xbc2fbb5f 0x00000062
```

```
[ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:1.30.0:check (default-cli) on project 
org.openhab.binding.amazonechocontrol: The following files had format violations:
[ERROR]     src/main/java/org/openhab/binding/amazonechocontrol/internal/WebSocketConnection.java
[ERROR]         @@ -553,7 +553,7 @@
[ERROR]          ········}
[ERROR]          
[ERROR]          ········byte[]·encodePing()·{
[ERROR]         -············//·MSG·0x00000065·0x0e414e47·f·0x00000001·0xbc2fbb5f·0x00000062·
[ERROR]         +············//·MSG·0x00000065·0x0e414e47·f·0x00000001·0xbc2fbb5f·0x00000062
[ERROR]          ············this.messageId++;
[ERROR]          ············String·msg·=·"MSG·0x00000065·";·//·Message-type·and·Channel·=·CHANNEL_FOR_HEARTBEAT;
[ERROR]          ············msg·+=·this.encodeNumber(this.messageId)·+·"·f·0x00000001·";
[ERROR] Run 'mvn spotless:apply' to fix these violations.
```